src/main/java/com/znsio/teswiz/runner/AppiumDriverManager.java

```diff
@@ -313,17 +313,16 @@ private static void terminateAndroidAppOnDevice(String appPackageName,
         ApplicationState applicationState = null;
         try {
             LOGGER.info(String.format("Terminate app: %s", appPackageName));
-            // todo - fix for appium 2.0
-//            applicationState = appiumDriver.queryAppState(appPackageName);
+            AndroidDriver androidDriver = (AndroidDriver) appiumDriver;
```

Comment:
Is this specific to AndroidDriver only? What about for iOS or Windows? If that is not supported, then this method will fail when running ios or Windows tests